### ISSUE TITLE
[Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.04.4)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#51878]
#### VFS team name
DGIB - My Education Benefits

#### VFS product name
VA Education Letters

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### Issue Details
I'm a little unclear on what the intention of the COE Decision Letter Update info alert is, and how that might impact a user's choices. This may be an issue, or it may not be.

If the information in that message could impact whether a user would choose to download their letter, then the message should come before the download link. With its current placement, some users (especially screen reader users and screen magnification users) may never reach that message if they just follow the download link once they get to it.

If that info alert isn't important to a user deciding whether to download their letter, then you can probably disregard this issue.

### VA.gov Experience Standard
[Category Number 02, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

### Platform Recommendation
As I said above, I'm not sure how critical this info alert is -- you know your product better than me, so that's your call! If the alert is relevant to deciding whether to download the letter, it should be moved either above the "Education decision letter" heading (and the alert heading changed to an H3), or above the "Letter available for you to download" heading (and alert heading kept at H2).

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by btruongafs** [comment]
Content recommendations from https://github.com/department-of-veterans-affairs/va.gov-team/issues/52549 were implemented to resolve this issue. Changes have been deployed to staging. Closing out.
